Source/Utils/WarpXUtil.cpp

```diff
@@ -247,9 +247,32 @@ Parser makeParser (std::string const& parse_function, amrex::Vector<std::string>
 
     Parser parser(parse_function);
     parser.registerVariables(varnames);
-    ParmParse pp_my_constants("my_constants");
+
     std::set<std::string> symbols = parser.symbols();
     for (auto const& v : varnames) symbols.erase(v.c_str());
+
+    // PICMI specifies inputs under this name, through which expressions
+    // can be provided for arbitrary variables. This potentially
+    // includes variable names that match physical or mathematical
+    // constants, in case the user wishes to enforce a different
+    // system of units or some form of quasi-physical behavior in the
+    // simulation. Thus, this needs to override any built-in
```

Comment:
I am not sure if this is confusing.
The constants only apply for parsed functions and inputs. This does not change the constants used in, e.g., the field solver.